default.nix

```diff
@@ -8,6 +8,33 @@ in
     dart = callPackage ./pkgs/dart {};
     flutter = callPackage ./pkgs/flutter {};
     flutter-patch = callPackage ./pkgs/flutter-patch {};
+    all-tools = cli-tools ++ clojure-tools ++ desktop-tools;
+    cli-tools = with pkgs; [
+      awscli
+      fzf
+      gettext
+      jq
+      kubectl
+      nodejs-10_x
+      nss
+      nssTools
+      openfortivpn
```

Comment:
Do we need this as we use the docker image made by you?

Reply:
I don't use that Docker image anymore, using the `openfortivpn` is a superior solution IMO since it is opensource and works much better in unstable connections.